pilot/SettingsViewController.swift

```diff
@@ -19,6 +19,8 @@ class SettingsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        self.navigationController?.navigationItem.largeTitleDisplayMode = .never
```

Comment:
What does this do?

Reply:
So if a navigation controller has large titles set to true then all the views lower in the navigation stack will also have large titles. By setting the largeTitleDisplay mode to never your saying "from this point forward I don't want  large titles." I thought this was appropriate since large titles are generally used for introduction views like "settings" or "messages." I mainly followed how Apple did it for their apps.

Reply:
Makes sense! That's cool that that's available as one line of code